k8s_test_harness/util/docker_util.py

```diff
@@ -12,3 +12,39 @@ def ensure_image_contains_paths(image, paths):
             ["docker", "run", "--rm", image, "ls", "-l", path],
             check=True,
         )
+
+
+def run_in_docker(image: str, check_exit_code: bool, *command: str):
```

Comment:
Nit: you could add kwargs for extra docker flags

Reply:
Added ``docker_args``, which can also cover single argument flags, not just pairs.